.github/workflows/www.yml

```
@@ -0,0 +1,39 @@
+on: push
+jobs:
+  build:
+    runs-on: ubuntu-18.04
+
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+
+      - name: Extract branch name
+        shell: bash
+        run: echo "##[set-output name=branch;]$(echo ${GITHUB_REF#refs/heads/})"
+        id: extract_branch
+
+      - name: docker login
+        env:
+          DOCKER_PASS: ${{ secrets.DOCKER_PASS }}
+          DOCKER_USER: ${{ secrets.DOCKER_USER }}
+        run: |
+          # docker login
+          docker version
+          docker login -u $DOCKER_USER -p $DOCKER_PASS
+
+      - name: docker build
+        env:
+          BRANCH: ${{ steps.extract_branch.outputs.branch }}
+          CF_API_KEY: ${{ secrets.CF_API_KEY }}
+          CF_EMAIL: ${{ secrets.CF_EMAIL }}
+          DOCKER_PASS: ${{ secrets.DOCKER_PASS }}
+          DOCKER_USER: ${{ secrets.DOCKER_USER }}
+          DOCKER_BUILDKIT: 1
+        run: |
+          export DOCKER_TAG=$(echo $BRANCH | tr -cd '[:alnum:]_')
+          cd packages/www
+          yarn run docker:build
+          yarn run docker:push
```

For consistency, is there any disadvantage with doing like this?
```suggestion
          yarn run lerna-run docker:build --scope=@livepeer.com/www
          yarn run lerna-run docker:push --scope=@livepeer.com/www
```

---

I forgot entirely why I did this until I committed this change - the `www` build skips the initial `yarn install` step entirely and jumps straight to the Docker parts. No need to download and link everything both inside and outside of the container; www doesn't have unit tests that could fail.

---

(So it doesn't have any idea what lerna is at this point)

---

Oh, I see. Yeah, it's nice to keep both a little more isolated for parallel speediness indeed. Maybe sometime we could even separate the "api" (docker?) build from the `main.yml` job as well, so it builds much faster. I bet we could get an isolated API build in ~4 minutes instead of the >~12 we have today.